components/src/atoms/price/Price.html

```diff
@@ -0,0 +1,24 @@
+<div class="price">
+  <!--@slot Custom old price slots (Named slots)-->
+  <slot name="oldPrice">
+    <del
+      v-if="oldPrice"
+      :aria-label="oldPriceAriaLabel"
+      class="price--old"
+    >
+      {{ oldPrice }}
+    </del>
+  </slot>
+  <!--@slot Custom special price slots (Named slots)-->
+  <slot name="specialPrice">
+    <ins
+      v-if="specialPrice"
+      :aria-label="specialPriceAriaLabel"
+      class="price--special"
+    >
+      {{ specialPrice }}
+    </ins>
+  </slot>
+  <!--@slot Default price slots-->
+  <slot/>
+</div>
```

Comment:
We don't have element or bloc for modifier at all http://prntscr.com/n7h6ue
My sugestion
```
<div class="price">
  <!--@slot Custom old price slots (Named slots)-->
  <slot name="oldPrice">
    <del
      v-if="oldPrice"
      :aria-label="oldPriceAriaLabel"
      class="price__value price__value--old"
    >
      {{ oldPrice }}
    </del>
  </slot>
  <!--@slot Custom special price slots (Named slots)-->
  <slot name="specialPrice">
    <ins
      v-if="specialPrice"
      :aria-label="specialPriceAriaLabel"
      class="price__value price__value--special"
    >
      {{ specialPrice }}
    </ins>
  </slot>
  <!--@slot Default price slots-->
  <slot/>
</div>
```